ckanext/canada/helpers.py

```diff
@@ -194,6 +194,8 @@ def portal_url():
     url = PORTAL_URL_DEFAULT_FR if h.lang() == 'fr' else PORTAL_URL_DEFAULT_EN
     return str(config.get(PORTAL_URL_OPTION, url))
 
+def search_url():
+    return str(config.get('ckanext.canada.adv_search_url_fr')) if h.lang() == 'fr' else str(config.get('ckanext.canada.adv_search_url_en'))
```

Comment:
Do we really need separate url configuration for en/fr? The language is passed in the url in the template below, does it not work if the domain is wrong?

Reply:
right, forgot about the redirects

Reply:
Maybe newer browsers can handle this, but I seem to remember IE not being happy when it happened. I haven't tested in a while though.